README.md

```diff
@@ -40,4 +40,51 @@
 
  Your contribution will be greatly appreciated.
 
- See `docs/CONTRIBUTING.md` for further information.
\ No newline at end of file
+ See `docs/CONTRIBUTING.md` for further information.
+
+ Installation
+ ------------------------------
+
+ Follow these steps to set up and run the Coral Snake Game:
+
+ ### Prerequisites
+ 
+ - Python 3: Make sure you have Python 3 installed on your machine. You can download it from the official Python website.
```

Comment:
Add python download link using `download it from the official Python [website](https://www.python.org/downloads/)`